app/assets/javascripts/spree/backend/translations.js.coffee

```diff
@@ -30,7 +30,7 @@ $ ->
     display_locale_fields()
     false
 
-  $('#locale, #supported_locales_').select2({placeholder: Spree.translations['please_choose_language']})
+  $('#locale, #available_locales_').select2({placeholder: Spree.translations['please_choose_language']})
```

Comment:
I can't figure out the problem and how to solve it. The only select available in the view is available_locales. Without this change select2 doesn't get initialized on it. 

If refuse this change, then something needs to be done about the views... I need some help on figuring this out. I don't know how to fix it.